### PR TITLE
resolvconf: add page

### PR DESCRIPTION
**Pull request by DEBADRIBASAK**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/resolvconf.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# resolvconf
+
+> Manage nameserver information.
+> Acts as an intermediary between rograms that supply nameserver information and applications that use this information.
```
Comment (acuteenvy) [inline pages/linux/resolvconf.md:4]:
```suggestion
> Acts as an intermediary between programs that supply nameserver information and applications that use this information.
```

**Review by acuteenvy — COMMENTED** [review]
There are multiple implementations of `resolvconf`. This page documents Debian's `resolvconf`, but other distributions can use `openresolv`, so I don't know what to do here. It's also used on BSDs - the page should probably be moved to `common`.

File: pages/linux/resolvconf.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+> More information: <https://manpages.ubuntu.com/manpages/resolvconf.8.html>.
```
Comment (reinhart1010) [inline pages/linux/resolvconf.md:5]:
```suggestion
> More information: <https://manned.org/resolvconf.8>.
```

We use manned.org for vendor-neutral manpage documentation—See #5143.

File: pages/linux/resolvconf.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- Add or override a record (IFACE.PROG) and run the update scripts if updating is enabled:
```
Comment (reinhart1010) [inline pages/linux/resolvconf.md:7]: Are you sure that the record is written as `IFACE.PROG` (or `interface<dot>program` as far as I can understood)? Some manpages refer this as `interface<dot>protocol`, like `dhcp` and `ppp`.

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Here aresome reviews from mine.

**Comment by reinhart1010** [comment]
> There are multiple implementations of `resolvconf`. This page documents Debian's `resolvconf`, but other distributions can use `openresolv`, so I don't know what to do here. It's also used on BSDs - the page should probably be moved to `common`.

@acuteenvy just read yohr comments. After skim-reading this with the [manned.org version (based on BSD)](https://manned.org/resolvconf.8) I agree that this should be moved to `common`. Can you also provide the `openresolv` manpage version to compare with?

**Comment by acuteenvy** [comment]
https://manned.org/resolvconf documents openresolv.

> HISTORY
> This implementation of resolvconf is called openresolv

File: pages/linux/resolvconf.md (hunk 3 above)
Comment (reinhart1010) [reply pages/linux/resolvconf.md:7]: Oh I see that this is caused by the difference between Debian or openresolv.

File: pages/linux/resolvconf.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`resolvconf -a {{IFACE.PROG}}`
+
+- Delete the record IFACE.PROG amd run the update scripts if updating is enabled:
```
Comment (kbdharun) [inline pages/linux/resolvconf.md:11]:
```suggestion
- Delete the IFACE.PROG record and run the update scripts if updating is enabled:
```

File: pages/linux/resolvconf.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`resolvconf -d {{IFACR.PROG}}`
+
+- Just run the update scripts if updating is enabled:
+
+`resolvconf -u`
+
+- Set the flag indicating whether resolvconf should run update scripts when invoked with -a, -d or -u:
```
Comment (kbdharun) [inline pages/linux/resolvconf.md:19]:
```suggestion
- Set the flag indicating whether `resolvconf` should run update scripts when invoked with `-a`, `-d` or `-u` flags:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by DEBADRIBASAK** [comment]
> There are multiple implementations of `resolvconf`. This page documents Debian's `resolvconf`, but other distributions can use `openresolv`, so I don't know what to do here. It's also used on BSDs - the page should probably be moved to `common`.

What shall I do here. Will adding to linux suffice or shall I move it to common?

File: pages/linux/resolvconf.md (hunk 2 above)
Comment (DEBADRIBASAK) [reply pages/linux/resolvconf.md:5]: This link https://manned.org/resolvconf.8 documents `operesolve` implementation and I am documenting Debian's implementation. Shall I change it or is there any way to add all the variations on the same page?

File: pages/linux/resolvconf.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (DEBADRIBASAK) [reply pages/linux/resolvconf.md]: Addressed.

File: pages/linux/resolvconf.md (hunk 4 above)
Comment (DEBADRIBASAK) [reply pages/linux/resolvconf.md]: Addressed.

File: pages/linux/resolvconf.md (hunk 5 above)
Comment (DEBADRIBASAK) [reply pages/linux/resolvconf.md]: Addressed.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by reinhart1010 — APPROVED** [review]

**Pull request merged**